polar-core/src/resource_block.rs

```diff
@@ -298,7 +355,7 @@ impl ResourceBlocks {
         if let Some(declaration) = maybe_declaration {
             Ok(declaration)
         } else {
-            let msg = format!("Undeclared term {} referenced in rule in the '{}' resource block. Did you mean to declare it as a role, permission, or relation?", declaration, resource_name);
+            let msg = format!("Undeclared term {} referenced in rule in '{}' resource block. Did you mean to declare it as a role, permission, or relation?", declaration.to_polar(), resource_name);
```

Comment:
`.to_polar()` usually (always?) defaults to the `Display` impl — you shouldn't need to add an explicit call to `.to_polar()` when you're already formatting the thing as `{}` (which uses the thing's `Display` impl)

Reply:
✂️ `.to_polar()`